workshops/templates/workshops/person_merge_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "workshops/_page.html" %}
+
+{% load crispy_forms_tags %}
+{% load selectable_tags %}
+
+{% block content %}
+
+    {% crispy person_merge_form %}
+
+{% endblock %}
```

Comment:
You're missing a block at the end that adds extra form media (CSS / JS):

```
{% block extrajs %}
{{person_merge_form.media}}
{% endblock extrajs %}
```

It will load JS and CSS for django-selectable lookup fields.